src/app/common/filter-bar/FilterBar.vue

```diff
@@ -4,111 +4,109 @@
     class="filter-bar"
     data-testid="filter-bar"
   >
-    <search>
-      <form
-        ref="$form"
-        @submit.prevent="change"
+    <form
+      ref="$form"
+      @submit.prevent="change"
+    >
+      <button
+        class="focus-filter-input-button"
+        title="Focus filter"
+        type="button"
+        data-testid="filter-bar-focus-filter-input-button"
+        @click="focusFilterInput"
       >
-        <button
-          class="focus-filter-input-button"
-          title="Focus filter"
-          type="button"
-          data-testid="filter-bar-focus-filter-input-button"
-          @click="focusFilterInput"
-        >
-          <span class="visually-hidden">Focus filter</span>
-
-          <span class="filter-bar-icon">
-            <FilterIcon
-              decorative
-              data-testid="filter-bar-filter-icon"
-              hide-title
-              :size="KUI_ICON_SIZE_30"
-            />
-          </span>
-        </button>
-
-        <label
-          :for="`${props.id}-filter-bar-input`"
-          class="visually-hidden"
-        >
-          <slot>
-            {{ placeholderAndLabelFallback }}
-          </slot>
-        </label>
-        <input
-          :id="`${props.id}-filter-bar-input`"
-          ref="filterInput"
-          v-model="currentQuery"
-          class="filter-bar-input"
-          type="search"
-          :placeholder="currentPlaceholder"
-          data-testid="filter-bar-filter-input"
-          name="s"
-          @focus="isShowingSuggestionBox = true"
-          @input="isShowingSuggestionBox = true"
-          @blur="closeSuggestionBoxIfCondition"
-          @search="(e: InputEvent) => {
-            const $el = e.target as HTMLInputElement
-            if($el.value.length === 0) {
-              clear(e)
-              isShowingSuggestionBox = true
-            }
-          }"
-        >
-
-        <div
-          v-if="isShowingSuggestionBox"
-          class="suggestion-box"
-          data-testid="filter-bar-suggestion-box"
-        >
-          <div class="suggestion-list">
-            <p
-              v-if="tokenizerError !== null"
-              class="filter-bar-error"
-            >
-              {{ tokenizerError.message }}
-            </p>
+        <span class="visually-hidden">Focus filter</span>
```

Comment:
Don't we use i18n?

Reply:
Yeah your right. Right now, kinda mostly for "machine to human" purposes tho. There are a couple of issues relevant for an FYI:

- https://github.com/kumahq/kuma-gui/issues/2814
- https://github.com/kumahq/kuma-gui/issues/2813

---

Separately, the code here comes from how it was previously, I just copy/pasted it elsewhere to move it for the fix, and additionally this PR was just a cherry-pick to port the fix we did on another branch so both commits match up in both `2.9.x` and `master`.

Would be fine moving these bits to our i18n strings here though if you think that would be better. lmk!

Reply:
One the one hand I like to have one topic per PR, but sometimes including a little fix on the way is fine 😄 anyways, since there is an issue for that and you just moved some things around, I am fine tackling this with #2813 👍